test/todolist/tags_test.exs

```diff
@@ -0,0 +1,64 @@
+defmodule Todolist.TagsTest do
+  use Todolist.DataCase
+
+  alias Todolist.Tags
+
+  describe "tags" do
+    alias Todolist.Tags.Tag
+
+    @valid_attrs %{name: "some name"}
```

Comment:
ExUnit also has a `setup` macro, which you can use globally or inside your `describe` section to set up various things. This way you don't need to create 10 module attrs for setting up dummy test data.

Reply:
May i know the advantage of using the `setup` macro as compared to using module attributes?

Reply:
This page might help: https://hexdocs.pm/ex_unit/ExUnit.Callbacks.html

Basically `setup` will merge the properties you set up in a "context" that is then piped into tests. This way you can have a "Global" setup and then a "describe-specific" setup and they get merged together etc.
It gives you more flexibility.

Reply:
I gave it a shot. Do take a look 😄